app/assets/javascripts/views/vacation_request.js

```diff
@@ -0,0 +1,123 @@
+App.Views.VacationRequest = Backbone.View.extend({
+  el: 'section',
+  template: JST['templates/vacation_request'],
+
+  initialize: function() {
+    this.$el.html( this.template() );
+    // Prepare handy access for the controls
+    this.$available_days  = this.$('#available_days');
+    this.$vacation_type   = this.$('select[name=vacation_type]');
+
+    this.available_vacations  = new App.Collections.AvailableVacations();
+    this.vacation_requests    = new App.Collections.VacationRequests();
+    this.listenTo( this.available_vacations,  'reset',  this.render );
+    this.listenTo( this.vacation_requests,    'all',  this.logg );
```

Comment:
Why? Interpreter does not care about that, but I do. I have no coding style guides that I have to follow.